tests/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ if (NOT NO_STATIC_ANALYSIS)
 endif ()
 
 # C options
-set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD 99)
```

Comment:
If we claim support for C11, we should test against it, too. I recommend removing this statement completely and adding an extra parameter to the test matrix generation function `gen_test_matrix` below specifying the version of the standard. See example in https://github.com/pavel-kirienko/o1heap/blob/master/tests/CMakeLists.txt